SPECS/rsyslog/rsyslog.spec

```diff
@@ -128,6 +128,10 @@ find %{buildroot} -type f -name "*.la" -delete -print
 %check
 %make_build check
 
+%pre
+groupadd --system syslog
```

Comment:
I think we want a `Requires(pre): shadow-utils` to accompany this. I checked the full dependency graphs and the only extra dependency that gets included due to `shadow-utils` is `libsemange`.